paasta_tools/api/views/instance.py

```diff
@@ -47,17 +45,20 @@ def marathon_job_status(mstatus, client, job_config):
         mstatus['error_message'] = error_msg
         return
 
-    if marathon_tools.is_app_id_running(app_id, client):
-        app = client.get_app(app_id)
-        deploy_status, _ = get_marathon_app_deploy_status(app, app_id, client)
-        mstatus['deploy_status'] = MarathonDeployStatus.tostring(deploy_status)
+    mstatus['app_id'] = app_id
+    mstatus['expected_instance_count'] = job_config.get_instances()
+
+    deploy_status, deploy_delay = marathon_tools.get_marathon_app_deploy_status(app_id, client)
```

Comment:
following on from the comment I left at the ``get_marathon_app_deploy_status`` function, just make this
```
deploy_status = marathon_tools.get_marathon_app_deploy_status(app_id, client)
```
you already have the code on line 60 to get the delay (so, in the code's current form, this is calculated twice?)